src/__tests__/OPLS.test.js

```diff
@@ -316,36 +316,38 @@ describe('OPLS utility functions', () => {
 });
 
 describe('OPLS', () => {
-  it('test nComp = 1', () => {
-    let x = new Matrix(iris);
-
-    // let trainTestLabels = require('../../data/trainTestLabels.json');
-    // let options = { trainTestLabels, nComp: 1, folds };
-
-    let cvFolds = getCrossValidationSets(7, { idx: 0, by: 'trainTest' });
-
-    let options = { cvFolds, nComp: 1 };
+  // eslint-disable-next-line jest/no-commented-out-tests
```

Comment:
Why are those test commented ? As worse you coud use 'describe.skip' but if there is a test I would expect it pass.

Reply:
This is because with the new version of OPLS calculates as many components as it needs and stops automatically. Those tests has a defined number of components to calculate.